.github/workflows/daily.yml

```diff
@@ -282,6 +282,76 @@ jobs:
       if: true && !contains(github.event.inputs.skiptests, 'cluster')
       run: ./runtest-cluster --config io-threads 4 --config io-threads-do-reads yes ${{github.event.inputs.cluster_test_args}}
 
+  test-ubuntu-reclaim-cache:
```

Comment:
there is a failure in the Daily, can you take a look?
https://github.com/redis/redis/actions/runs/4169023701/jobs/7216495785